### ISSUE TITLE
MongoDB setup instructions

### ISSUE THREAD

**Issue by gniezen** [tidepool-org/tidepool-org.github.io#70]
Hi there,

I've had some trouble getting everything up and running on my local machine due to my MongoDB setup not working as expected. Blip runs but shows a *Blip is stuck and isn't doing what you want it to do. We're sorry for the trouble. Blip will attempts to send the details to our server* error when setting up a new user.

While `get_current_tidepool_repos.sh` does a good job if checking if all the prerequisites are installed, there are some additional setup instructions for a new MongoDB installation that could be included as part of the [Getting Started](http://developer.tidepool.io/starting-up-services/) manual?

Basically, after you've installed MongoDB with `brew install mongodb`, you still need to do the following:

- Run `sudo mkdir -p /data/db`
- Set permissions for `/data/db` to local user using `sudo chown <your username> /data/db`

**Comment by jebeck** [comment]
Thanks for this @gniezen. We are having trouble keeping our docs up to date... :( Any chance you'd be willing to submit a PR on the relevant doc, probably [this](https://github.com/tidepool-org/tidepool-org.github.io/blob/master/_docs/starting-up-services.md) would be the best place for it?
